hazelcast/test/src/CMakeLists.txt

```diff
@@ -5,13 +5,10 @@ FILE(GLOB_RECURSE HZ_TEST_HEADERS "./*h")
 
 set(GOOGLETEST_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/hazelcast/test/googletest/googletest/include)
 
-SET(HZ_EXE_NAME clientTest_${HZ_LIB_TYPE}_${HZ_BIT})
+add_executable(client_test ${HZ_TEST_SOURCES} ${HZ_TEST_HEADERS})
```

Comment:
should we change min re cmake version the same for all cmake files? was it forgotten here or was it needed to be different?

Reply:
All version are set to 3.10.